change/@ni-nimble-tokens-387d48fd-bf9e-4c3d-8d40-e7c58687f8e6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Since this is a larger visual change it would probably be good to show the diffs to Brandon to make sure he doesn't have any concerns. That could happen after the PR completes (e.g. at our team meeting) if he's not immediately available.

Reply:
I ran the changes by Brandon. He thinks they look good.